10/RequestReply.md

```diff
@@ -103,15 +103,40 @@ NOTE All GROUPED REQUESTS should be logged once with the same information as the
 
 ### Connection management
 
-* The Reqest-Reply protocol SHALL manage client connection.
+The Request-Reply protocol SHALL manage client connections to Device Servers.
 
 #### Connection establishement
 
+* To send any REQUESTS to a Device, a Client MUST connect to the Device Server manging the Device.  
+* A Device Server SHALL wait for client connections on one or multiple network endpoints.
+* The network endpoint SHALL define \<network_address\> and \<tcp_port\>, where:
+
+... abnf
+ network_address = ip_address / host_name
+ tcp_port = num-val
+...
+
+* A Client MAY connect any Device Server on one of Device Server's endpoints.
+* If a Client is running on the same host as a Device Server, the client MAY use other than
+  network endpoints like operating system inter-process communication mechanism.
+* A Client MAY get information about Device Server's endpoints:
+  * From the Tango Database service, if the service is available,
```

Comment:
I would rather say that the Tango Database specify the way to get the connection information. Other way have to be specify by the Tango Database RFC.

Reply:
I look on [the RFC-6](https://github.com/tango-controls/rfc/blob/draft/6/Database.md#exported-devices) and I thinkit is worth to add that the Device Server SHALL call DbExportDevice to publish the endpoint and DbUnExportDevice to unpublish.

Reply:
Good point! In this case we should emphasis the role of the Device Server to use the  Request-Reply to register the new connection.